Sources/KeyboardShortcuts/SwiftUI+Extensions.swift

```diff
@@ -0,0 +1,123 @@
+// Created by Neil Clayton on 17/07/2024.
+// Using ideas by mbenoukaiss, https://github.com/sindresorhus/KeyboardShortcuts/issues/101
+
+
+import Foundation
+import SwiftUI
+import KeyboardShortcuts
+import Carbon
+
+// Provides a SwiftUI like wrapper func, that feels the same as the normal SwiftUI keyboardShortcut
+@available(macOS 11.0, *)
+extension View {
+	@ViewBuilder
```

Comment:
This is moot

Reply:
Except that in my case it's being included by code that is also compiled before v11. Unless I misunderstand?

Reply:
The comment is on `@ViewBuilder`